src/main/java/be/polyscripts/contactmanagerapp/dto/AuthenticationRequest.java

```diff
@@ -0,0 +1,35 @@
+package be.polyscripts.contactmanagerapp.dto;
+
+public class AuthenticationRequest {
+    private String username;
+    private String password;
+
+    public String getUsername() {
+        return username;
+    }
+
+    public void setUsername(String username) {
+        this.username = username;
+    }
+
+    public String getPassword() {
+        return password;
+    }
+
+    public void setPassword(String password) {
+        this.password = password;
+    }
+
+    public AuthenticationRequest(String username, String password) {
+        super();
```

Comment:
why super() here?
you can just use @AllArgsConstructure

Reply:
done !